### PR TITLE
Stations & Observations forms

### PR DESCRIPTION
**Pull request by ya-web**

### CODE REVIEW

File: assets/js/app.js [hunk 1]
```diff
@@ -7,12 +7,628 @@
...
+function onFileEvent() {
+    let droppedFiles = false,
+        $picture = $('.upload-zone .upload-input');
+    let isAdvancedUpload = function() {
+        let div = document.createElement('div');
+
+        return (('draggable' in div) || ('ondragstart' in div && 'ondrop' in div)) && 'FormData' in window && 'FileReader' in window;
+    }();
+
+    if (isAdvancedUpload) {
```
Comment (kstefanini) [inline assets/js/app.js:364]: Ce serait pas isAdvancedUpload() plutôt ?

Comment (ya-web) [reply assets/js/app.js:364]: si

Comment (kstefanini) [reply assets/js/app.js:364]: J'avais mal lu, mon mauvais.

File: src/DisplayData/Station/StationDisplayData.php [hunk 2]
```diff
@@ -120,7 +197,7 @@ private function getStationObsWithPictures(): array
 
     private function sortObsByDAte(Observation $obsA, Observation $obsB)
     {
-        return $obsB->getDateObs() <=> $obsA->getDateObs();
+        return $obsB->getObsDate() <=> $obsA->getObsDate();
```
Comment (kstefanini) [inline src/DisplayData/Station/StationDisplayData.php:200]: Just getDate of Observation is fine : $obs->getDate() is pretty simple to read and understand. 
No need for getDateObs or getObsDate

File: src/Entity/Observation.php [hunk 3]
```diff
@@ -97,15 +107,39 @@ public function setPicture(?string $picture): self
     }
 
 
-    public function getDateObs(): ?\DateTimeInterface
+    public function getObsDate(): ?\DateTimeInterface
```
Comment (kstefanini) [inline src/Entity/Observation.php:110]: Property should be `date` instead of `obs_date`

File: src/Entity/Observation.php [hunk 4]
```diff
@@ -43,6 +43,16 @@ class Observation
      */
     private $obs_date;
 
+    /**
+     * @ORM\Column(type="boolean")
+     */
+    private $is_missing;
```
Comment (kstefanini) [inline src/Entity/Observation.php:49]: Don't use snake case. Should be `isMissing`

**Pull request merged**